app/views/DefaultBranchListPage.scala.html

```diff
@@ -79,28 +104,29 @@ <h1>Default Branch</h1>
             </td>
             <td class="teamNames monospaced" id="row@{i}_teamNames">
                 @if(repo.teamNames.length > 0 && repo.teamNames.length <= 6) { @for(name <- repo.teamNames) {
-                            @if(name == repo.teamNames.last) { @name } else { @name, }
+                    @if(name == repo.teamNames.last) { @name } else { @name, }
                         }
                     } else if (repo.teamNames.length > 6)  {
-                        <i>Various Teams</i>
+                        <i title="@for(name <- repo.teamNames) { @if(name == repo.teamNames.last) { @name } else { @name,}} ">Various Teams</i>
```

Comment:
you can just use `mkString`: `repos.map(_.teamNames).mkString(",")`
Same above.